webpack.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  // Uncomment when debugging
+  // devtool: 'eval-cheap-module-source-map',
```

Comment:
:joy: :sob:

Reply:
Better to set this automatically when `WATCH=1`?

Reply:
Yes, or maybe always for `yarn dev`? Seems like we should have source maps always? I wanted to check with you on this. But I've found myself needing it quite a bit to get work done, so I'm hesitant to have to keep stashing my change away.